archivessnake/edit_notes.py

```diff
@@ -82,16 +81,13 @@ def main():
     """Main function, which is run when this script is executed"""
     start_time = time.time()
     parser = get_parser()
-    args = parser.parse_args()
-    global aspace
-    aspace = ASpace()
-    global writer
+    client = ASpace().client
+    spreadsheet_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), OUTPUT_FILENAME)
     writer = csv.writer(open(spreadsheet_path, "w"))
-    create_spreadsheet(["Box Number", "Archival Object URI"])
-    process_tree(args, aspace.repositories(AS_REPOSITORY).resources(args.resource_id))
+    create_spreadsheet(writer, ["Collection Title", "Finding Aid Number", "URI", "Ref ID", "Object Title", "Box Number"])
+    process_tree(client.get(f'/repositories/2/resources/{(args.resource_id)}').json(), parser)
```

Comment:
Here you are passing a JSON instance of a resource, but `process_tree` is expecting to be called with the `client` itself.